scripts/superdesk/editor2/editor.js

```diff
@@ -834,19 +834,25 @@ angular.module('superdesk.editor2', [
                         }
                     });
                 }
+                var debouncedInit = _.debounce(init, 250);
```

Comment:
The image cropping view is now using debounce, does it cause protractor to randomly fail @petrjasek ?